content/en/docs/concepts/security/security-checklist.md

```diff
@@ -206,6 +206,15 @@ SELinux is only available on Linux nodes, and enabled in
 [some Linux distributions](https://en.wikipedia.org/wiki/Security-Enhanced_Linux#Implementations).
 {{< /note >}}
 
+## Logs and auditing
+
+- [ ] Audit logs, if enabled, are protected from general access
```

Comment:
nit: Use a period at the end of sentence

Is there a question about the wording of the second item or why the second item belongs in the checklist?
I found the wording somewhat unclear. How could you improve or clarify the second item?



The `/logs` API is disabled (you are running kube-apiserver with
  `--enable-logs-handler=false` and not the default setting where it is enabled).

Reply:
I reworded, how's this?